### PR TITLE
chore(ujust): Improve setup-virtualization user experience

### PR DESCRIPTION
**Pull request by Zeglius**
- Remove "run after reboot" requirement.
- `disable-virtualization` will now remove any virtualization related packages/kernel args regardless of virt-manager being installed,

### CODE REVIEW

**Review by HikariKnight — DISMISSED** [review]

**Comment by HikariKnight** [comment]
LGTM

File: system_files/desktop/shared/usr/share/ublue-os/just/84-bazzite-virt.just [hunk 1]
```diff
@@ -41,29 +41,30 @@ setup-virtualization ACTION="":
...
-          sudo systemctl disable --now libvirtd 2> /dev/null
+      (
+        virt_test=$(rpm-ostree status -v --jsonpath '.deployments[0].packages')
+        if [[ ${virt_test} == *virt-manager* ]]; then
+          echo "Installing QEMU and virt-manager..."
+          rpm-ostree install -y virt-manager edk2-ovmf qemu
+          rpm-ostree kargs \
+          --append-if-missing="kvm.ignore_msrs=1" \
+          --append-if-missing="kvm.report_ignored_msrs=0"
+          sudo systemctl enable bazzite-libvirtd-setup.service \
```
Comment (HikariKnight) [inline system_files/desktop/shared/usr/share/ublue-os/just/84-bazzite-virt.just:52]: Think you might have forgotten to disable this when you disable virtualization, other than that it looks good.

Comment (HikariKnight) [inline system_files/desktop/shared/usr/share/ublue-os/just/84-bazzite-virt.just:52]: Think you might have forgotten to disable this when you disable virtualization, other than that it looks good.

Comment (Zeglius) [reply system_files/desktop/shared/usr/share/ublue-os/just/84-bazzite-virt.just:52]: Yeah missed that one, fixed with commit c03ed222fcd23784bd2712cc2b677155310216b8

**Review by HikariKnight — APPROVED** [review]
LGTM now

**Pull request merged**